### ISSUE TITLE
Request- folder is always downloads

### ISSUE THREAD

**Issue by Sotapanna** [cuixiping/Save-Image-as-Type#27]
Thank you for updating this!

Any chance of getting the most recent used folder remembered and used? 

Always saving in downloads, adds a step I'd skip if I could...

**Comment by CameosVatican** [comment]
Yes, I'd vote for this too please, @cuixiping...

I stopped my saves going to Downloads a long time ago, and switched to another key folder that I use the most often just by adjusting my Chrome settings, but yes, there are a handful I use very regularly. 

It would certainly be helpful for the app to either remember _most recent,_ or for there to be a small drop-down of eg the four most recent (as my Mac already does in a native save).  Not sure if this could be a sub-option of the context menu even, as I've seen other extensions add a layer of options this way – but of course those options are fixed by the _developer_, not the user – you couldn't possibly know everyone's favourite local folder options... 😉

@Sotapanna: what I'm doing as a temp workaround if there are _several_ saves I need to make, is to just save them all in eg 'Downloads', then batch-move them across to the destination folder.  I also batch-rename and can batch-convert eg 100 images to .png by using the Mac native context menu in Finder – I expect PCs have a similar feature if that's what you use...

**Comment by Sotapanna** [comment]
@CameosVatican 

Yick..  can you not on a mac create a symlink to a directory outside the structure?  i.e. a shortcut to another folder?  
That's what I did-   made two bogus symlinks to the two directories I wanted and placed them in the downloads folder.

**Comment by HorseTears** [comment]
Yes, this a great suggestion from OP. Defintely would appreciate the extension remembering the most recent download folder.
